avm/res/databricks/access-connector/main.json

```diff
@@ -5,8 +5,8 @@
   "metadata": {
     "_generator": {
       "name": "bicep",
-      "version": "0.24.24.22086",
-      "templateHash": "5801242046314845466"
+      "version": "0.25.3.34343",
```

Comment:
This would cause a new version to be published. We are saving that to publish all modules with the new `--with-source` option. Could you please manually revert this change?

Reply:
Done

Reply:
@clintgrove thanks for owning this module. Could you please review & approve this PR?